docs/demo/demo.md

```diff
@@ -49,6 +49,24 @@ export LLVM_SRCDIR=~/llvm-project
 export LLVM_INSTALLDIR=~/llvm-install
 ```
 
+### Depot Tools
```

Comment:
This shouldn't be needed. Jiri downloads CIPD and installs it in `${FUCHSIA_SRCDIR}/.jiri_root/bin/cipd`. If you follow the instructions at https://fuchsia.dev/fuchsia-src/get-started/get_fuchsia_source#set-up-environment-variables and set the environment variables as suggested, `cipd` should be in your `PATH`.

Reply:
Thanks for catching that. Should be fixed after the latest force push.